### DISCUSSION TITLE
[@types/three] Property 'TextGeometry' does not exist ?

### DISCUSSION THREAD

**Discussion by pdsuwwz** [DefinitelyTyped/DefinitelyTyped#57617] [Issues with a @types package] [answered]
* "three": "^0.135.0"
* "@types/three": "^0.135.0",

```ts
console.log(THREE.TextGeometry) // Property 'TextGeometry' does not exist
```

![image](https://user-images.githubusercontent.com/19891724/145148256-f7c9da00-e591-4e52-b59c-f42bff2a4058.png)

[a comment by an automated account is not shown]

**Comment by Methuselah96** [comment, marked as answer]
`TextGeometry` was removed from the core in r133. It is now located in `examples/jsm/geometries/TextGeometry.js`. See the [migration guide](https://github.com/mrdoob/three.js/wiki/Migration-Guide#132--133).

**Reply by pdsuwwz** [reply]
> `TextGeometry` was removed from the core in r133. It is now located in `examples/jsm/geometries/TextGeometry.js`. See the [migration guide](https://github.com/mrdoob/three.js/wiki/Migration-Guide#132--133).

Thank you for your help! I will try to see if this works.
